Sources/App/Settings/Connection/ConnectionURLViewController.swift

```diff
@@ -289,6 +295,39 @@ final class ConnectionURLViewController: HAFormViewController, TypedRowControlle
                 }
             }
         }
+
+        form +++ Section(footer: L10n.Settings.ConnectionSection.AlwaysFallbackInternal.footer)
+            <<< SwitchRow(RowTag.alwaysFallbackToInternalURL.rawValue) {
+                $0.title = L10n.Settings.ConnectionSection.AlwaysFallbackInternal.title
+                $0.value = server.info.connection.alwaysFallbackToInternalURL
+
+                $0.cellUpdate { cell, _ in
+                    cell.switchControl.onTintColor = .red
+                }
+
+                $0.onChange { [weak self] row in
+                    if row.value ?? false {
+                        let alert = UIAlertController(
+                            title: L10n.Settings.ConnectionSection.AlwaysFallbackInternal.Confirmation.title,
+                            message: L10n.Settings.ConnectionSection.AlwaysFallbackInternal.Confirmation.message,
+                            preferredStyle: .actionSheet
```

Comment:
Action sheet needs a source view on iPad or it crashes: https://stackoverflow.com/a/34762275

Reply:
Good catch, thanks